trade-finance-manager-ui/templates/case/case.njk

```diff
@@ -20,6 +21,10 @@
     {{caseSummary.render(componentData)}}
   {% endif %}
 
+  {% if successMessage %}
+   {{ successBanner.render(message = successMessage) }}
```

Comment:
I always forget how nunjucks macros work - can we simplify this? If this is a stupid suggestion ignore & resolve :) 

```suggestion
   {{ successBanner.render(successMessage) }}
```

Reply:
It can, good spot! For some reason I thought nunjucks didn't like that, but seems all good!